fenix/app/src/main/java/org/mozilla/fenix/search/SearchDialogFragment.kt

```diff
@@ -118,6 +119,8 @@ class SearchDialogFragment : AppCompatDialogFragment(), UserInteractionHandler {
 
     private val qrFeature = ViewBoundFeatureWrapper<QrFeature>()
     private val speechIntent = Intent(RecognizerIntent.ACTION_RECOGNIZE_SPEECH)
+    private val isPrivate: Boolean
+        get() = (requireActivity() as HomeActivity).browsingModeManager.mode.isPrivate
```

Comment:
nit: If this property isn't needed for multiple functionalities then it's value can be inlined in the call site.